### ISSUE TITLE
Post-Template Readme popup doesn't work

### ISSUE THREAD

**Issue by darrenj** [microsoft/BotFramework-Composer#7253]
## Describe the bug

Some templates post creation have manual steps the developer must follow. For example, a complex template like the Enterprise Template has manual configuration steps that need to be done.

Without this it's hard for developers to figure out why something doesn't work (see #7252). Some of the next steps are solved for some templates today - e.g. templates using QnA and LUIS the rocket ship menu makes mandatory.

Prioritization is difficult on this, if the EA template can't figure out a workaround to step people getting stuck this might end up being a p0.

## Version

1.4.0-nightly.237625

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Create an EA template 

## Expected behavior

- Modal popup (or similar) showing the README contents? Might get in the way too much?
- Alternatively and better perhaps have the rocket ship have a required next step of reviewing Next Steps before starting the Bot (just for the Enterprise Template at least for now)

**Comment by darrenj** [comment]
Spoke to @elaine-chang and sounds like another issue might be kicking around to show the readme on the rocket ship menu which would work

**Comment by elaine-chang** [comment]
Yes, Issue [6993](https://github.com/microsoft/BotFramework-Composer/issues/6993) is the racket menu change UI change issue that we proposed a bunch of changes including having a link to readme. 

Adding @lauren-mills and @thomas-chung as FYI.

**Comment by elaine-chang** [comment]
Since the readme issue (#5779) mentioned in #6993 was already closed, I created #7323 to track this the readme as the minimal fix of this. The ideal fix of adding the steps into the rocket menu is tracked with #7325.

**Comment by Dewain27** [comment]
Yes, Duplicate
